common/src/main/java/com/wynntils/models/rewards/type/TomeType.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.rewards.type;
+
+import com.wynntils.utils.StringUtils;
+import java.util.Optional;
+
+public enum TomeType {
+    WEAPON(true, true),
+    ARMOUR(true, true),
+    GUILD("Allegiance", true, false),
```

Comment:
Do guild tomes really have a set name? I have a few and I don't remember this line

Reply:
I honestly don't know. I don't have a single Tome. I have copied all this code straight out from the old code which I believe @P0keDev added.

Reply:
I checked and it seems to be true. I guess I forgot..

Reply:
Yes, all tomes of a certain type follow the same name pattern - although I will admit this code isn't the cleanest representation of that